lib/editing/editor_codemirror.css

```diff
@@ -111,12 +111,20 @@ li.CodeMirror-hint-active {
 }
 
 .type-getter::before {
-  content: "⇽";
+  content: "\21FD";
   background: radial-gradient(#e1C2ff, #cdb7e8);
 }
 
 .type-setter::before {
-  content: "⇾";
+  content: "\21FE";
+  background: radial-gradient(#e1C2ff, #cdb7e8);
+}
+
+.type-getter_and_setter::before {
+  content: "\21FD\21FE";
```

Comment:
Does `21FF` work here? Drawing two chars over top of each other - which is what I assume is happening here - will cause anti-aliased text to be less distinct.